packages/overlay/overlay-trigger.md

```diff
@@ -0,0 +1,127 @@
+```
+
+## Examples
+
+Here a default `<overlay-trigger>` manages content that is triggered by click and "hover" interactions.
+
+```html
+<style>
+    overlay-trigger {
+        flex: none;
+    }
+
+    .tooltip {
+        background-color: var(--spectrum-global-color-gray-800);
+        color: var(--spectrum-global-color-gray-50);
+        padding: 4px 10px;
+        font-size: 10px;
+    }
+</style>
+<overlay-trigger id="trigger" placement="bottom" offset="6">
+    <sp-button variant="primary" slot="trigger">Button popover</sp-button>
+    <sp-popover dialog slot="click-content" direction="bottom" tip>
+        <div class="options-popover-content">
+            <sp-slider
+                value="5"
+                step="0.5"
+                min="0"
+                max="20"
+                label="Awesomeness"
+            ></sp-slider>
+            <sp-button>Press me</sp-button>
+        </div>
+    </sp-popover>
+    <sp-tooltip slot="hover-content" delayed>Tooltip</sp-tooltip>
+    <sp-popover slot="longpress-content" tip>
+        <sp-action-group
+            selects="single"
+            vertical
+            style="margin: calc(var(--spectrum-actiongroup-button-gap-y,var(--spectrum-global-dimension-size-100)) / 2);"
+        >
+            <sp-action-button>
+                <sp-icon-magnify slot="icon"></sp-icon-magnify>
+            </sp-action-button>
+            <sp-action-button>
+                <sp-icon-magnify slot="icon"></sp-icon-magnify>
+            </sp-action-button>
+            <sp-action-button>
+                <sp-icon-magnify slot="icon"></sp-icon-magnify>
+            </sp-action-button>
+        </sp-action-group>
+    </sp-popover>
+</overlay-trigger>
+```
+
+### Click content only
+
+This example only delivers content via the click interaction and leverages both `placement` and `type` attributes to customize the visual relationship of the content to the page and its position in the tab order.
+
+```html
+<overlay-trigger placement="top" type="replace">
+    <sp-button slot="trigger">Overlay Trigger</sp-button>
+    <sp-popover slot="click-content" open>
+        <sp-dialog size="small">
+            <h2 slot="heading">Click content</h2>
+            An &lt;overlay-trigger&gt; can be used to manage either or both of
+            the "click" and "hover" content slots that are made available. Here,
+            content is only addressed to
+            <code>slot="click-content"</code>
+            ...
+            <sp-button
+                slot="button"
+                onclick="javascript: this.dispatchEvent(new Event('close', {bubbles: true, composed: true}));"
+            >
+                I understand
+            </sp-button>
+        </sp-dialog>
+    </sp-popover>
+</overlay-trigger>
+```
+
+### "Hover" content only
+
+Hover content can be customized via the `placement` attribute, however it not allowing secondary interactions the `type` attribute will not further customize its delivery.
```

Comment:
```suggestion
Hover content can be customized via the `placement` attribute. However, it not allowing secondary interactions the `type` attribute will not further customize its delivery.
```
Can we think of an alternate way to say this? I make some suggestions, but I'm not sure exactly what the intended meaning is so I could be off. 
"however it not allowing secondary interactions"
-  However, by not allowing...
- However, it doesn't allow secondary interactions, so the `type` attribute will not further customize its delivery.